개팬다/HEEA/BOJ_1003_2_.py

```diff
@@ -0,0 +1,32 @@
+z=[1,0]
+for _ in range(39):
+    z.append(-1)
+o=[0,1]
+for _ in range(39):
+    o.append(-1)

```

Comment:
```suggestion
z=[-1]*41
z[0], z[1] = 1, 0
o=[-1]*41
o[0], o[1] = 0, 1
```
초기화는 취향 차이, 알고는 있겠지만 이런거도 된다.